src/jdk.hotspot.agent/share/classes/sun/jvm/hotspot/SALauncher.java

```diff
@@ -86,7 +86,7 @@ private static boolean commonHelp(String mode, boolean canConnectToRemote) {
         System.out.println("          or  jhsdb " + mode + " --core ./core.1234 --exe ./myexe");
         if (canConnectToRemote) {
             System.out.println("          or  jhsdb " + mode + " --connect debugserver");
-            System.out.println("          or  jhsdb " + mode + " --connect id@debugserver");
+            System.out.println("          or  jhsdb " + mode + " --connect id@debugserver:1234");
```

Comment:
You change here makes it look like if you specify `id@` then you also need to specify the port. I'd suggest also including the original line that just has `id@debugserver`.

Reply:
I reverted the original line in new commit.

We can also specify port number without `id@` (e.g. `--connect debugserver:1234`). Is it ok not to describe on help message? IMHO it is not good to describe all patterns because it might be verbosely.

Reply:
> On the other hand, maybe you could go with just the id@debugserver:1234 example and leave the other 3 off. The syntax does clearly show the id and port are optional.

Agree, I left `id@debugserver:1234` only in new commit.